raiden/tests/conftest.py

```diff
@@ -12,9 +12,11 @@
 from ethereum import slogging
 from ethereum.tools.keys import PBKDF2_CONSTANTS
 
+from raiden.exceptions import RaidenShuttingDown
 from raiden.tests.fixtures import *  # noqa: F401,F403
 
 gevent.get_hub().SYSTEM_ERROR = BaseException
+gevent.get_hub().NOT_ERROR = (gevent.GreenletExit, SystemExit, RaidenShuttingDown)
```

Comment:
Is this only about printing/logging? Docs seem to not say anything more: http://www.gevent.org/gevent.hub.html#gevent.hub.Hub.NOT_ERROR

Reply:
Humm, I had the impression it stopped propagating the exception.

Reply:
I would be interested to know if it does

Reply:
This is the only usage of that variable:

https://github.com/gevent/gevent/blob/c7efa2b7583445d03b0598fa06ba26e77bcb6d53/src/gevent/hub.py#L582-L583

so yes, it's only used to control printing